src/sentry/mediators/sentry_app_installations/destroyer.py

```diff
@@ -41,6 +46,21 @@ def _destroy_service_hooks(self):
         for hook in hooks:
             service_hooks.Destroyer.run(service_hook=hook)
 
+    def _destroy_api_tokens(self):
```

Comment:
@MeredithAnya Where is `_destroy_api_tokens` used?

Reply:
oh whoops that never gets called - gotta add that in